### PR TITLE
added product listing page

### PR DESCRIPTION
**Pull request by vivek-guda**
Added
->Card component with add to cart and wishlist buttons
->Sidebar with filter options
ecom-product-page

### CODE REVIEW

File: src/components/Card/CartCard.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import "./ProductCard.css";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
```
Comment (srv-pranjal) [inline src/components/Card/CartCard.js:3]: You can create an index.js file inside context folder and export all the contexts from it. This way you will be able to import all contexts from a single file.

File: src/components/Card/ProductCard.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  width: fit-content;
+  width: -moz-fit-content;
+  position: absolute;
+  bottom: 1rem;
+  left: 1rem;
+  z-index: 3;
+}
+
+.star-icon {
+  color: #169690;
```
Comment (srv-pranjal) [inline src/components/Card/ProductCard.css:37]: Try to use variable for colors and background colors.

File: src/pages/Products/Products.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      let res = false;
+      for (let i = 0; i < filterState.filterByCategory.length; i++) {
+        res |= category === filterState.filterByCategory[i];
+      }
+      return res;
+    }
+    categoryData = categoryData.filter((item) => filterCategory(item.category));
+
+    //filterByBrand
+    let xdata = categoryData;
```
Comment (srv-pranjal) [inline src/pages/Products/Products.js:37]: Try to use meaningful names for the variables.

**Review by srv-pranjal — APPROVED** [review]
UI looks pretty clean.
Github suggestion
 - You can deploy your branch on netlify, it creates a deploy preview and helps the reviewer.

Left some suggestions, rest everything looks good.

File: src/pages/Products/Products.js (hunk 3 above)
Comment (vivek-guda) [reply src/pages/Products/Products.js:37]: ok sure

**Pull request merged**